docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md

```diff
@@ -151,7 +151,7 @@ async initializePreOrderPayment() {
       data: {
         type: 'payments',
         attributes: {
-          quote: QUOTE_DATA,
+          quote: QUOTE_DATA, // Must contain the `grandTotal` field and should be as complete as possible
```

Comment:
so where is grandTotal?
 https://github.com/spryker-projects/spa-checkout-glue-with-stripe/blob/main/src/public/assets/main.js#L214-L229

```
    const requestData = {
      data: {
        type: 'payments',
        attributes: {
          quote: {
            customer: this.customer,
            billingAddress: this.address,
            shippingAddress: this.address,
            currency: {
              code: 'EUR',
            },
            store: {
              name: 'DE',
            },
            totals: {
              taxTotal: {
                amount: 100,
              },
            }
          },
          payment: {
            amount: this.grandTotal,
            paymentMethodName: 'stripe',
            paymentProviderName: 'stripe',
          },
          preOrderPaymentData: {
            "transactionId": this.transactionId,
          },
        },
      },
    };
```

Reply:
Looks like that this is not the problem :( The grandTotal will be added from the payment object to the orderData that gets passed to the App.

Then we have a different problem.

Closing this one.